src/routes/author.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const authorController = require('../controllers/author');
+
+const router = express.Router();
+
+router.post('', authorController.addAuthor);
+router.get('', authorController.getAllAuthors);
```

Comment:
Although there is nothing wrong with this, it is considered best practice to put a / between the quotations in the first parameter if there is no specific route after the base route
```
router.post('/', authorController.addAuthor);
router.get('/', authorController.getAllAuthors);
```